core/controllers/editor.py

```diff
@@ -697,12 +698,12 @@ class ExplorationStatisticsHandler(EditorHandler):
     def get(self, exploration_id, exploration_version):
         """Handles GET requests."""
         try:
-            exp_services.get_exploration_by_id(exploration_id)
+            exploration = exp_services.get_exploration_by_id(exploration_id)
```

Comment:
You might need to call this current_exploration or similar, since the exploration domain object for an old version will differ from a domain object representing the current version of the exploration.

That said, instead of passing in `exploration` I think it makes more sense in this case to pass in exp_id and a list of state_names. Don't pass in stuff that's not needed.